### ISSUE TITLE
Request: Launch minimized

### ISSUE THREAD

**Issue by captn3m0** [yuezk/GlobalProtect-openconnect#47]
I would like to launch the client application at startup, but minimized to tray. A `gpclient --start-minimized` flag would be nice that would let me configure the application for autostart without being distracting.

**Comment by yuezk** [comment]
Good idea, I will do research on that.

**Comment by lenonk** [comment]
+1

**Comment by atharen** [comment]
Gave it a quick look, I presume it would need to set `QMainWindow` to `Qt::WindowMinimized` using [`QWidget::setWindowState`](http://qt-project.org/doc/qt-5.0/qtwidgets/qwidget.html#setWindowState) ([_Source_](https://stackoverflow.com/a/3599489))

To implement it, it would require a `QMainWindow` object, which [from a search](https://github.com/yuezk/GlobalProtect-openconnect/search?q=QMainWindow), this would be [`GPClient`](https://github.com/yuezk/GlobalProtect-openconnect/blob/9d6ec84c14de8c55dd154a6d95a4283e6577ba47/GPClient/gpclient.h#L17).

---

Now thinking about it, not sure if minimize is the intended functionality here; 
basically what I imagine is that the application is running in the background, with no window visible, _i.e. the GPClient dialog is closed_.

To view the window, one would need to open it from the AppIndicator.

**Comment by captn3m0** [comment]
For reference, https://github.com/flameshot-org/flameshot/ does this in the default mode, with a `flameshot gui` command that launches the GUI as well.

**Comment by yuezk** [comment]
So many people waiting for it. I will try to implement it.

**Comment by yuezk** [comment]
Released in 1.4.5. `gpclient --start-minimized`.
